packages/cli/src/commands/oracle/remove-expired-reports.ts

```diff
@@ -0,0 +1,43 @@
+import { CeloContract } from '@celo/contractkit'
+import { BaseCommand } from '../../base'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class RemoveExpiredReports extends BaseCommand {
+  static description =
+    'Remove expired oracle reports for a specified token (currently just Celo Dollar, aka: "StableToken")'
+
+  static args = [
+    {
+      name: 'token',
+      required: true,
+      default: CeloContract.StableToken,
+      description: 'Token to remove expired reports for',
+      options: [CeloContract.StableToken],
+    },
+  ]
+  static flags = {
+    ...BaseCommand.flags,
+    from: Flags.address({
+      required: true,
+      description: 'Address of the account removing oracle reports',
+    }),
+  }
+
+  static example = [
+    'remove-expired-reports StableToken --from 0x8c349AAc7065a35B7166f2659d6C35D75A3893C1',
+    'remove-expired-reports --from 0x8c349AAc7065a35B7166f2659d6C35D75A3893C1',
+  ]
+
+  async run() {
+    const res = this.parse(RemoveExpiredReports)
+    this.kit.defaultAccount = res.flags.from
+    const sortedOracles = await this.kit.contracts.getSortedOracles()
+    const reports = await sortedOracles.getReports(res.args.token)
+    const tokenAddress = await this.kit.registry.addressFor(res.args.token)
```

Comment:
With the current master version of the `SortedOracles` wrapper, you can skip this address lookup.